archunit/src/test/java/com/tngtech/archunit/library/ArchitecturesTest.java

```diff
@@ -218,6 +219,55 @@ public void layered_architecture_combines_multiple_ignores() {
         assertThat(layeredArchitecture.evaluate(classes).hasViolation()).as("result has violation").isFalse();
     }
 
+    @Test
+    public void layered_architecture_where_all_classes_are_covered_fully() {
+        JavaClasses classes = new ClassFileImporter().importClasses(
+                FirstAnyPkgClass.class,
+                SecondThreeAnyClass.class);
+
+        LayeredArchitecture architecture = layeredArchitecture()
+                .layer("One").definedBy("..first..")
+                .layer("Two").definedBy("..second..")
+                .whereAllClassesAreCovered();
+
+        assertThat(architecture.getDescription()).contains("where all classes are covered");
```

Comment:
Similar to my comment in `Architectures`, I'm not sure if we actually should adjust the description. In particular we should probably make it consistent to #273 where as far as I saw the description is not adjusted. But this is pretty much the same thing, a configuration of *how* the layered architecture should be tested against the input, not about the layered architecture itself.

Reply:
Interesting. Omitting this from the description seems counter-intuitive but I'll have a play and try and convince myself one way or the other.

Reply:
Sure! To me the description is like a report of the architecture spec. But I have no strong opinion, if this should be part of the spec or not. I just have a strong opinion that #273 and #278 should do it the same way, one way or the other :wink:

Reply:
I've dropped the descriptions for now in f431a75. Perhaps later I'll open a separate merge request adding descriptions for both related cases when I've had chance to make a convincing argument.